openedx/core/djangoapps/course_date_signals/tests.py

```diff
@@ -122,6 +122,38 @@ def test_sequence_with_graded_and_ungraded_assignments(self):
             sequence = self.store.get_item(sequence.location)
             self.assertCountEqual(_gather_graded_items(sequence, 5), expected_graded_items)
 
+    def test_sequence_with_ora_and_non_ora_assignments(self):
+        """
+        _gather_graded_items should not set a due date for ORA problems
+        """
+        with self.store.bulk_operations(self.course.id):
+            sequence = ItemFactory(parent=self.course, category="sequential")
+            vertical = ItemFactory(parent=sequence, category="vertical")
+            sequence = self.store.get_item(sequence.location)
```

Comment:
Is this line necessary?

Reply:
just double checked and nope! Will delete!